README.md

```diff
@@ -71,12 +71,21 @@ $ brew update && brew install cassowary
 ```
 
 ### Docker  
-You can run **cassowary** directly by using it's Docker image:  
+You can run **cassowary** directly by using its official Docker image:  
 
 ```bash
 $ docker run rogerw/cassowary:v0.14.0 cassowary run -u http://www.example.com -c 1 -n 10
 ```
 
+For local development:
+
+```bash
+$ env GOOS=linux go build -o dist/docker
```

Comment:
GOOS=linux go build -o dist/docker/cassowary cmd/cassowary/*.go

Reply:
thanks for correction! it is updated accordingly